environments/icds-cas/postgresql.yml

```diff
@@ -119,6 +119,66 @@ dbs:
       name: warehouse
       host: pgwarehouse0
       django_migrate: True
+    - name: commcarehq_p10
```

Comment:
I want to start the databases with p10, because I want our hostname, database and their hostname numbers to all match to reduce confusion